src/main/java/org/example/Main.java

```diff
@@ -2,7 +2,71 @@
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Привет Мир!");
-        System.out.println("ДЗ Готово!");
+// Первая задача
+        int personAge = 16;// количество лет
+        if (personAge >= 18)//Условие совершеннолетия//
+        {
+            System.out.println("Человек совершеннолетний");
+        } else System.out.println("Человеку меньше 18, он не достиг совершеннолетия, нужно немного подождать");
+// Вторая задача
+        byte outsideWeather = 7; // Указываем температуру на улице
+        if (outsideWeather <= 5) {
+            System.out.println("На улице - " + outsideWeather + " , нужно надеть шапку.");
+        } else System.out.println("На улице - " + outsideWeather + ". Сегодня тепло, можно идти без шапки.");
+// Третья задача
+        short actualSpeed = 50;//Указываем текущую скорость
+        short speedLimit = 60;//Указываем ограничение скорости
+        if (actualSpeed > speedLimit) {
+            System.out.println("Скорость превышена");
+        } else System.out.println("Превышения скорости нет");
+//Четвертая задача
+        byte actualAge = 2;//Указываем текущий возраст
+        if (actualAge > 2 && actualAge <= 6)//Возраст хождения в ДС
+        {
+            System.out.println("Текущий возраст - " + actualAge + ". Надо ходить в детский сад");
+        } else if (actualAge >= 7 && actualAge <= 17)//Возраст хождения в школу
+        {
+            System.out.println("Текущий возраст - " + actualAge + ". Надо ходить в школу");
+        } else if (actualAge >= 18 && actualAge <= 24)//Возраст хождения в университет
+        {
+            System.out.println("Текущий возраст - " + actualAge + ". Ваше место в университете");
+        } else if (actualAge > 24)//Возраст хождения на работу
+        {
+            System.out.println("Текущий возраст - " + actualAge + ". Бегом на завод!");
+        } else {
+            System.out.println("Человек еще слишком маленький, малышам надо сидеть дома с мамой.");
+        }//Вывод результата, если возраст меньше 2
+// Пятая задача
+        byte actualAge2 = 15;//Указываем текущий возраст
```

Comment:
нет смысла создавать новую переменную, можно просто использовать старую
```suggestion
        actualAge = 15;//Указываем текущий возраст
```